### ISSUE TITLE
performance regression ?

### ISSUE THREAD

**Issue by sdementen** [ZoomerAnalytics/xlwings#568]
I have the impression that there is some performance regression in the new version vs the old one I was using. Is there any benchmark that tracks the evolution of the performance of xlwings ?

**Comment by fzumstein** [comment]
No, we've always been focusing on making it easy vs. making it efficient, so it's possible that the performance got impacted by the recent changes. If you can come up with a sample spreadsheet and performance difference between two versions, we can look into it.

**Comment by kaiaeberli** [comment]
@sdementen : since which version have you observed this performance regression? I would like to downgrade as I have the same issue. In my experience, version 0.7 was still fine.

**Comment by sdementen** [comment]
I can't confirm this easily

**Comment by kaiaeberli** [comment]
Hi fzumstein,

I finally got around to creating a self contained sample of the slowness occurring in version 0.10 compared to 0.7. Please see attached.

[ExcelPython.zip](https://github.com/ZoomerAnalytics/xlwings/files/744481/ExcelPython.zip)

Regards,
Kai

**Comment by fzumstein** [comment]
OK great thanks for that. I ll try to replicate the issue on my end and let you know!
